### DISCUSSION TITLE
How can I use remark-footnotes with remark-deflist?

### DISCUSSION THREAD

**Discussion by brti-dev** [remarkjs/remark#670] [Q&A] [answered]
One may encounter the following error when attempting to use footnotes and deflist plugins together:

```
Error: Cannot handle unknown node 'footnoteReference'
```

This is how I'm attempting to use them together in my Next.js app:

```javascript
import remark from 'remark'
import remarkHtml from 'remark-html'
import remarkDeflist from 'remark-deflist'
import remarkFootnote from 'remark-footnotes'

async function markdownToHtml(md) {
    const processedContent = await remark()
        .use(remarkFootnote, { inlineNotes: true })
        .use(remarkDeflist)
        .use(remarkHtml)
        .process(md)

    return processedContent.toString()
}
```

removing either `footnotes` or `deflist` from the mix works fine! Is there a better way to use these together than the function as it is?

**Comment by wooorm** [comment, marked as answer]
I suggest asking the `remarkDeflist` folks.

What they’re doing does look a bit weird btw 🤔
